snuba/datasets/storages/sessions.py

```diff
@@ -122,7 +122,10 @@ def process_query(self, query: Query, request_settings: RequestSettings) -> None
     query_processors=[MinuteResolutionProcessor(), TableRateLimit()],
     mandatory_condition_checkers=[OrgIdEnforcer(), ProjectIdEnforcer()],
     stream_loader=build_kafka_stream_loader_from_settings(
-        processor=SessionsProcessor(), default_topic=Topic.SESSIONS,
+        processor=SessionsProcessor(),
+        default_topic=Topic.SESSIONS,
+        commit_log_topic=Topic.COMMIT_LOG,
```

Comment:
Sessions should have its own commit log topic rather than using the one from events and sharing parts of the pipeline. It could add unnecessary load to subscriptions and post process forwarders.

Reply:
Sure that makes sense! Do I achieve that by just using a different `Topic` enum?

Reply:
Yes, just add a new enum member to Topic and use it here

Reply:
Done